realm/realm-library/src/androidTest/java/io/realm/rule/TestRealmConfigurationFactory.java

```diff
@@ -66,7 +68,9 @@ protected void before() throws Throwable {
     protected void after() {
         try {
             for (RealmConfiguration configuration : configurations) {
-                Realm.deleteRealm(configuration);
+                if (!Realm.deleteRealm(configuration)) {
```

Comment:
Uh... we talked about this, the only case returns `false` right now is the file doesn't exist. `deleteRealm` itself will throw ISE when the Realm is still opened. right?